bindings/python/src/converters.cpp

```diff
@@ -132,11 +132,16 @@ struct dict_to_map
         dict o(borrowed(x));
         std::map<T1, T2> m;
 
-        list iterkeys = (list)o.keys();
-        int const len = int(boost::python::len(iterkeys));
-        for (int i = 0; i < len; i++)
+        const object dictkeys = o.keys();
```

Comment:
is this a better (version agnostic) way of doing the same thing?
``stl_input_iterator<object> begin(o), end;``
and iterate over them